src/Model/ROI.py

```diff
@@ -26,6 +26,19 @@ def rename_roi(rtss, roi_id, new_name):
     return rtss
 
 
+def delete_list_of_rois(rtss, rois_to_delete):
```

Comment:
```suggestion
def delete_list_of_rois(rtss : Dataset, rois_to_delete : List[str]):
```

while typing is loose in Python, providing it in signatures is useful  
you'll also need to add

from typing import List

Reply:
I've actually been using strong typing in my personal projects outside Onko because it makes it a lot clearer, but I felt it was better to have a consistent style in Onko than my preferred style.

I think a discussion could be had regarding updating the style guide in redmine to encourage strong typing (I would definitely be on the side supporting strong typing).